.github/renovate.json

```diff
@@ -46,6 +46,26 @@
       "description": "Lucene dependencies should be skipped",
       "matchPackageNames": ["org.apache.lucene"],
```

Comment:
Another issue is that this rule does not work. The bot opened a lucene upgrade PR on Feb 27 even if this rule has been in place since June 2023. https://github.com/apache/solr/pull/2309

Reply:
I tried to fix this by using `matchPackagePrefixes` instead of `matchPackageNames`, which matches package name exactly, which would be e.g. `org.apache.lucene:lucene-core`, i.e. including group and artifact.